README.md

```diff
@@ -162,6 +162,17 @@ the `encryption_*` and `ssl_*` options
 directly to the constructor as keyword arguments, even though it is
 required when they are placed in the environment file.
 
+Creating PAM or Native Credentials File (.irodsA)
+-------------------------------------------------
+
+```Two free functions exist for creating encoded authentication files.
+    irods.client_init.write_credentials_with_native_password
+    irods.client_init.write_credentials_with_pam_password
```

Comment:
Should this be `write_pam_credentials_to_secrets_file`?

Reply:
done